src/Microsoft.AspNetCore.Mvc.Razor/RazorPageOfT.cs

```diff
@@ -20,6 +20,9 @@ public abstract class RazorPage<TModel> : RazorPage
     {
         private IModelMetadataProvider _provider;
 
+        [RazorInject]
+        private ExpressionTextCache _expressionTextCache { get; set; }
```

Comment:
- This is still a property and should be named `ExpressionTextCache`.
- Was placed correctly before. Add `private` members last.